### ISSUE TITLE
Poppins font is not 100% centered.

### ISSUE THREAD

**Issue by ozamed** [google/fonts#3457]
Hey!

I'm working with React Native using Expo and I have some problems with Poppins font. The font is not 100% aligned center. To be sure I used couple of other fonts and they aligned perfectly, even React Native's default font. Still I'm not sure is this a feature of the Poppins font family but I'm here to learn. Here's some detail to describe my problem.

This is my buttons with React Native's default font family.
![1](https://user-images.githubusercontent.com/61123774/120216680-01deb480-c240-11eb-9dbc-6b3f21e21e7a.png)

This is my buttons with Poppins font family.
![2](https://user-images.githubusercontent.com/61123774/120216799-25a1fa80-c240-11eb-8254-269eece47816.png)

This is my style for button component.
```js
height: 60
marginBottom: 20
marginRight: 20
marginLeft: 20,
borderRadius: 10,
backgroundColor: colors.indigo,
justifyContent: 'center',
alignItems: 'center',
```

This is my style for text:
```js
fontSize: 24,
fontFamily: 'Poppins_600SemiBold',
color: colors.white,
```

**Comment by xconsau** [comment]
Hello Amed,

The vertical centering of a font depends upon it's vertical metrics (Ascender, descender, Line gap, etc.). In order to change this, the font needs to be updated with new metrics. Changing vertical metrics may also affect existing websites, considering that the Poppins font is popular and has a large user base. Furthermore, the update may not arrive in few days. It may take months or more in clearing the entire update procedure.

In the meantime, I would recommend adding a padding to the button.

<pre>buttonText: {
    fontSize: 24,
    fontFamily: 'Poppins_600SemiBold',
    color: colors.white,
    paddingTop: 5
  }</pre>

Hope this works.

**Comment by mrrenaud** [comment]
Shouldn't we reopen this ? This is not really a satisfying solution. Depending on the scenario, we will have to add different padding.

**Comment by jmessy919** [comment]
Yeah - this issue is also quite prominent when working with spreadsheets in Excel. Poppins is so flagrantly out of vertical alignment that I actually have to select the top-most vertical align option to achieve a somewhat normal layout, and even then it's not _actually_ centered.

Lovely, lovely font - but this is a major usability concern across desktop & web applications. I suppose one could open the font int a local editor and make vertical metric changes, but I don't know if that's some kind of font license violation...? Meh.,

Anyway, here's to hoping it gets figured out! 👍

**Comment by drookyn** [comment]
Try adding [includeFontPadding](https://reactnative.dev/docs/text-style-props#includefontpadding-android) prop to your Text or change it globally by adding it to your [styles.xml](https://developer.android.com/guide/topics/resources/style-resource).

```
<resources>
  <!-- ... -->
 
  <style name="AppTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
      <!-- ... -->
      <item name="android:includeFontPadding">false</item>
      <!-- ... -->
  </style>

  <!-- ... -->
</resources>
```

Works for me using React-Native@0.72.3

**Comment by lesthoward** [comment]
> Try adding [includeFontPadding](https://reactnative.dev/docs/text-style-props#includefontpadding-android) prop to your Text or change it globally by adding it to your [styles.xml](https://developer.android.com/guide/topics/resources/style-resource).
> 
> ```
> <resources>
>   
>  
>   <style name="AppTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
>       
>       <item name="android:includeFontPadding">false</item>
>       
>   </style>
> 
>   
> </resources>
> ```
> 
> Works for me using React-Native@0.72.3

@drookyn This works to me, it seems like some fonts add padding in Android

**Comment by samjusaitis** [comment]
In regard to React Native, I encountered this issue when our Poppins line height was set to 1.2 (i.e. 120% of the font size). 
Setting it to 1.4 fixed it for us across Android and iOS.

**Comment by daviderickson** [comment]
Why is this closed?  We are trying to use it in powerpoint and it has all of these problems there (and more).
